.github/files/jetpack-staging-sites/run-atomic-deploy-e2e-tests.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+# This script is ran as part of the `UpdateJetpackStaging` TeamCity build.
+# It triggers an E2E test run against pre-defined atomic sites that have been updated with the newest Jetpack build.
+
+if [ -z "$SIGNATURE_KEY" ] || [ -z "$TRIGGER_URL" ]; then
```

Comment:
Let's prefer Bash syntax since we're using Bash.
```suggestion
if [[ -z "$SIGNATURE_KEY" || -z "$TRIGGER_URL" ]]; then
```